### DISCUSSION TITLE
Make ThemeToggleButton a switch instead of a radio

### DISCUSSION THREAD

**Discussion by inwardmovement** [withastro/docs#763] [Ideas]
I'd like to make `ThemeToggleButton` a switch instead of a radio, to make the whole area clickable instead of having to click on the right icon to toggle the corresponding theme.

![image](https://user-images.githubusercontent.com/9438102/173219927-07049306-097c-4ba1-8c9d-6e3585939b46.png)

If it sounds like a good idea, do we have a Preact switch component I could use as a template? I'm afraid I couldn't build it from scratch.

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

This no longer seems to be a viable discussion since we now use [Starlight](https://starlight.astro.build/). 

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!
